src/components/common/TemperatureBar.tsx

```diff
@@ -0,0 +1,30 @@
+import { Flex, Progress, Text, ProgressProps } from '@chakra-ui/react';
+
+type TemperatureBarProps = ProgressProps & {
+  size?: string;
+};
+
+const TemperatureBar = ({
+  value,
+  size = 'xs',
+  ...props
+}: TemperatureBarProps) => {
+  return (
+    <Flex align="center" maxWidth={value}>
+      <Progress
+        {...props}
+        colorScheme="green"
```

Comment:
```suggestion
        color="green.400"
```
이렇게 해도 적용이 되나요? 
적용이 되면 colorScheme는 color= green.500, green.400을 기본으로 하는게 좋을 것 같아요!!